CRM/Civigiftaid/Report/Form/Contribute/GiftAid.php

```diff
@@ -151,6 +151,10 @@ public function __construct() {
         $this->_columns['civicrm_value_gift_aid_submission']['fields']
         as $field => $values
       ) {
+        if($this->_columns['civicrm_value_gift_aid_submission']['fields'][$field]['name'] == 'amount'){
```

Comment:
Could we also unset `gift_aid_amount` field here? So, you should replace the condition in this `if` statement with something like:
```php
in_array($this->_columns['civicrm_value_gift_aid_submission']['fields'][$field]['name'], array('amount', 'gift_aid_amount'))
```